components/prism-gherkin.js

```diff
@@ -1,13 +1,79 @@
+	'table-body': {
+		'pattern': /(\n[ \t]*\|.+\|[^\n]*)+/g,
+		'lookbehind': true,
+		'inside': {
+			'outline': {
+				'pattern': /<[^>]+?>/g,
+				'alias': 'variable'
+			},
+			'td': {
+				'pattern': /[^\|]+/,
+				'alias': 'string'
+			},
+			'punctuation': {
+				'pattern': /(\|)/g
+			}
+		}
+	},
+	'table-head': {
+		'pattern': /(\n[ \t]*\|.+\|[^\n]*)/g,
+		'inside': {
+			'th': {
+				'pattern': /[^\|]+/,
+				'alias': 'variable'
+			},
+			'punctuation': {
+				'pattern': /(\|)/g
+			}
+		}
+	},
+	'atrule': /\n[ \t]+(那麼\<|那么\<|而且\<|當\<|当\<|并且\<|同時\<|同时\<|前提\<|假设\<|假設\<|假定\<|假如\<|但是\<|但し\<|並且\<|もし\<|ならば\<|ただし\<|しかし\<|かつ\<|하지만\<|조건\<|먼저\<|만일\<|만약\<|단\<|그리고\<|그러면\<|และ|เมื่อ|แต่|ดังนั้น|กำหนดให้|ಸ್ಥಿತಿಯನ್ನು|ಮತ್ತು|ನೀಡಿದ|ನಂತರ|ಆದರೆ|మరియు|చెప్పబడినది|కాని|ఈ పరిస్థితిలో|అప్పుడు|ਪਰ|ਤਦ|ਜੇਕਰ|ਜਿਵੇਂ ਕਿ|ਜਦੋਂ|ਅਤੇ|यदि|परन्तु|पर|तब|तदा|तथा|जब|चूंकि|किन्तु|कदा|और|अगर|و|هنگامی|متى|لیکن|لكن|فرض کیا|عندما|جب|ثم|تب|پھر|بفرض|بالفرض|با فرض|اور|اما|اگر|اذاً|آنگاه|כאשר|וגם|בהינתן|אזי|אז|אבל|Якщо|Һәм|Унда|Тоді|Тогда|То|Также|Та|Пусть|Припустимо, що|Припустимо|Онда|Но|Нехай|Нәтиҗәдә|Лекин|Ләкин|Коли|Когда|Когато|Када|Кад|К тому же|І|И|Задато|Задати|Задате|Если|Допустим|Дано|Дадено|Вә|Ва|Бирок|Әмма|Әйтик|Әгәр|Аммо|Али|Але|Агар|А також|А|Τότε|Όταν|Και|Δεδομένου|Αλλά|Þurh|Þegar|Þa þe|Þá|Þa|Zatati|Zaradi|Zakładając|Zadato|Zadate|Zadano|Zadani|Zadan|Za předpokladu|Za predpokladu|Youse know when youse got|Youse know like when|Yna|Yeah nah|Y'know|Y|Wun|Wtedy|When y'all|When|Wenn|WEN|Wanneer|wann|Vendar|Ve|vaj|Và|Und|Un|ugeholl|Too right|Toda|Thurh|Thì|Then y'all|Then|Tha the|Tha|Tetapi|Ter|Tapi|Takrat|Tak|Tada|Tad|Stel|Soit|Sipoze Ke|Sipoze ke|Sipoze|Siis|Și|Şi|Si|Sed|Se|Så|Quando|Quand|Quan|qaSDI'|Pryd|Privzeto|Potom|Potem|Pokud|Pokiaľ|Podano|Però|Pero|Pak|Oraz|Onda|Ond|Oletetaan|Og|Och|O zaman|Njuk|Niin|Nhưng|Nato|När|Når|Nanging|Nalikaning|Nalika|Mutta|Menawa|Men|Mas|Manawa|Maka|Majd|Mając|Mais|Maar|mä|Ma|Lorsque|Lorsqu'\<|Logo|Let go and haul|Lè sa a|Le sa a|Lè|Le|latlh|Lan|Kun|Kuid|Kui|Ko|Kiedy|Khi|Ketika|Kemudian|Keď|Když|Kaj|Kai|Kadar|Kada|Kad|Jeżeli|Jeśli|Ja|It's just unbelievable|Ir|In|I CAN HAZ|I|Ha|Givun|Givet|Given y'all|Given|Gitt|ghu' noblu'|Gegewe|Gegeven|Gegeben seien|Gegeben sei|Gdy|Gangway\!|Fakat|Étant donnés|Etant donnés|Étant données|Etant données|Étant donnée|Etant donnée|Étant donné|Etant donné|Et|És|Epi|Entonces|Entón|Então|Entao|En|Eğer ki|Ef|Eeldades|E|Ðurh|Duota|Dun|Donitaĵo|Donat|Donada|Do|Diyelim ki|Diberi|Dengan|Den youse gotta|DEN|De|Dato|Dați fiind|Daţi fiind|Dati fiind|Dati|Date fiind|Date|Data|Dat fiind|Dar|Dano|Dann|dann|Dan|DaH ghu' bejlu'|Dados|Dado|Dadas|Dada|Ða ðe|Ða|Cuando|Cho|Če|Ce|Cando|Când|Cand|Cal|But y'all|0|But at the end of the day I reckon|BUT|But|Buh|Blimey\!|Biết|Bet|Banjur|Bagi|Aye|awer|Avast\!|Atunci|Atesa|Atès|Apabila|Anrhegedig a|Angenommen|And y'all|And|Ananging|AN|An|an|Ampak|Amikor|Amennyiben|Ama|Als|Alors|Allora|Ali|Aleshores|Ale|Akkor|Ak|Adott|Ac|Aber|A zároveň|A tiež|A taktiež|A také|A|a|'ej|'ach|'a)[ \t]+/g,
```

Comment:
Why is the order of the languages different in this rules than in the previous ones?

It would probably easier if the order of the translations was always the same.